engine/runtime/src/main/java/org/enso/interpreter/runtime/ModuleSources.java

```diff
@@ -83,4 +86,48 @@ ModuleSources ensureSource(QualifiedName name) throws IOException {
   String getPath() {
     return file() == null ? null : file().getPath();
   }
+
+  /**
+   * Special support for reassigning locations of sources when running development version. When
+   * {@code -ea} is enabled, then we try to locate library files under the {@code
+   * distribution}/{@code lib} directories.
+   */
+  private static boolean alternativeFile(Source.SourceBuilder b, TruffleFile file) {
+    var root = file;
+    var stack = new LinkedList<String>();
+    while (root != null) {
+      if ("0.0.0-dev".equals(root.getName())) {
+        break;
+      }
+      stack.addFirst(root.getName());
+      root = root.getParent();
+    }
+    if (root == null) {
```

Comment:
If the source isn't located in `0.0.0-dev` directory, then perform no _alternation of the file_.